src/Chunk.ts

```diff
@@ -478,6 +479,8 @@ export default class Chunk {
 				} else if (resolution instanceof ExternalModule) {
 					node.renderFinalResolution(code, `"${resolution.id}"`);
 					// AST Node -> source replacement
+				} else if (resolution instanceof WasmModule) {
+					node.renderFinalResolution(code, `"${resolution.id}"`);
```

Comment:
Currently the generated code is
```js
Promise.resolve(require("/foobar/sventest/addTwo/src/addTwo.wasm")).then(({addTwo}) => {
  console.log("addTwo(1, 2)", addTwo(1, 2));
});


	function then(resolve) {
	  // ...
	}


var addTwo = /*#__PURE__*/Object.freeze({
  then: then
});
```

I believe I need to change the resolution to return the `addTwo` object.

Reply:
The inlining decision for dynamic imports is made here - https://github.com/rollup/rollup/pull/2113/files#diff-8c6f11349cc1bb370bdd557ed2bcbe7dR443. Can that be extended to a WASM check?

Reply:
Ok, this has been fixed.